deploy/charts/harvester/charts/cdi-operator/templates/serviceaccount.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  # NB(thxCode): name cannot be customized as below:
+  # name: {{ template "cdi-operator.fullname" . }}
+  # because the installation strategy dump action needs this ServiceAccount.
+  name: cdi-operator
+  labels:
+{{ include "cdi-operator.labels" . | indent 4 }}
+    app.kubernetes.io/component: metadata
```

Comment:
please resolve the newline, and prefer to use `rbac` of its component value, metadata sounds wired.

Reply:
I don't think ServiceAccount is a thing for RBAC, it is used to connect RBAC, it is a metadata described what the service account is.